testEnv/src/index.tsx

```diff
@@ -1,9 +1,8 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
 import App from './App';
+import { createRoot } from 'react-dom/client';
 
-ReactDOM.render(
-  <App />,
-  document.getElementById('root')
-);
-
+const rootElement = document.getElementById('root');
+// MEMO: https://blog.logrocket.com/how-to-use-typescript-with-react-18-alpha/
```

Comment:
公式リンクの方がいいかもしれませんが、任せます。

```suggestion
// MEMO: https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html#updates-to-client-rendering-apis
```

Reply:
↓の型ガード(?)の事を補足していたから、このリンクでも良いかなぁ
`if (!rootElement) throw new Error('Failed to find the root element');`